code/index.html

```diff
@@ -1,22 +1,234 @@
+          <h2>News of the Day</h2>
+          <h3 class="main-subtitle">No More Capybara Hunting in Colombia</h3>
+          <p>... or almost. Colombian law enforcement arrested an illegal
+            <a href="https://en.wikipedia.org/wiki/Capybara" target="_blank">capybara</a> hunting network in the region
+            of
+            Casanare. The organization was not only hunting these adorable hamster-like creatures, but they were also
+            selling their meat in awful sanitary conditions! Their arrest is part of a recent initiative for the
+            conservation of the species in the South American country.
+          </p>
+          <p>
+            Capybaras (or chiguiros, as they are called in Colombia) are a protected species under Colombian law. Sport
+            hunting was banned decades ago, and commercial hunting is only allowed for three months a year. Obtaining a
+            license to hunt and sell capybara isn't easy, either! So it's good to hear that authorities are making sure
+            people are actually leaving chiguiros alone. Long live the world's biggest rodent!
+          </p>
+          <form
+            action="https://www.elespectador.com/judicial/cae-presunta-red-de-caza-ilegal-de-chiguiro-que-vendia-su-carne-en-malas-condiciones/"
+            method="get" target="_blank"><button type="submit" class="cta-button read-more">Read
+              more</button>
+          </form>
+        </div>
+      </div>
     </section>
+    <section class="featured-news" id="jump-to-featured">
+      <div class="featured-news-container">
+        <h2 class="featured-title">Featured news</h2>
+        <main class="grid">
+          <!-- A grid containing several secondary news -->
+          <article>
+            <img class="featured-image" src="./resources/img/site-content/chino-rocha-P69rwd0vyHw-unsplash.jpg"
+              alt="elderly couple wedding">
+            <div class="featured-single">
+              <h3>Renewed vows</h3>
+              <p>Elderly couple finally gets to celebrate 50th anniversary postponed by the pandemic.</p>
+              <a class="read-more-link"
+                href="https://www.today.com/news/couple-renews-vows-live-today-after-pandemic-postponed-50th-anniversary-t224083"
+                target="_blank">Read
+                more</a>
+            </div>
+          </article>
+          <article>
+            <img class="featured-image" src="./resources/img/site-content/rachel-barkdoll-Ns0wpr-DO-g-unsplash.jpg"
+              alt="woman playing baseball">
+            <div class="featured-single">
+              <h3>Baseball Dream</h3>
+              <p class="featured-paragraph">70 year-old NY Yankees fan serves as a bat girl (her lifetime dream).</p>
+              <a class="read-more-link"
+                href="https://www.today.com/news/yankees-fan-serve-bat-girl-60-years-after-being-tuSample photorned-t223991"
+                target="_blank">Read
```

Comment:
Not sure I follow... Do you mean placing the <a class="read-more-link"> right outside the <div class="featured-news-text">?

Reply:
This is just a suggestion since it is a stylistic preference. So might be the wrong way to go for your project. But i would probably put all the buttons on the bottom of the div so that they are aligned in a line next to eachother no matter how much content you have in the box. Because now you have the buttons on a lot of different vertical levels which might give it a less clean appearance. But I Want to be clear that this has Nothing to do with your code at all, this is just a personal preference and is  not necessarily the right way to go with your Content.

Reply:
Ooh now I get it! It's actually a great suggestion, it's true that the layout of the articles could benefit from aligning the `read-more`. Thanks for the idea, I'll try it!